DSharpPlus/Enums/Interaction/ComponentType.cs

```diff
@@ -36,12 +36,28 @@ public enum ComponentType
         /// </summary>
         Button = 2,
         /// <summary>
-        /// A select menu.
+        /// A select menu consisting of options.
         /// </summary>
-        Select = 3,
+        StringSelect = 3,
```

Comment:
breaking change, so we might want to consider keeping the old name.

alternatively we can just release 4.3.0 with a breaking change if we announce it properly, that also works.

also, the documentation change is quite redundant and should be clarified.

Reply:
Keep it compatible.

Reply:
Compatible, but incorrect. Furthermore, the last stable release (4.2.0) was released 6 months ago (04/15).